src/sonic-gnmi/.gitignore

```diff
@@ -0,0 +1,10 @@
+build/
```

Comment:
please create new repo sonic-gmni under sonic-net org and use submodule approach.